tools/automation/generators/relnoted/relnotetester-nohighlights.yml

```diff
@@ -0,0 +1,58 @@
+product: Postgres Distributed
+version: 5.6
+date: 15 October 2024
+intro: |
+ Reassuring text about this release
+relnotes:
+- relnote: SDW support with Streaming Transaction
+  component: bdr
+  component_version: 5.6
+  details: |
+    One of the main advantages of streaming is that the WAL sender sends the partial transaction before it commits, which 
+    reduces replication lag. Now, with streaming support, the WAL decoder does the same thing, but it streams to the LCRs
+    segments. Eventually, the WAL sender will read the LCRs and mimic the same behavior of streaming large transactions
+    before they commit. This provides the benefits of SDW, such as reduced CPU and disk space, as well as the benefits of
+    streaming, such as reduced lag and disk space, since ".spill" files are not generated.
+
+    The WAL decoder always streams the transaction to LCRs, but based on downstream requests, the WAL sender either 
+    streams the transaction or just mimics the normal `BEGIN..COMMIT` scenario.
+
+    In addition to the normal LCRs segment files, we create streaming files with the starting names 
+    `STR_TXN_<file-name-format>` and `CAS_TXN_<file-name-format>` for each streamed transaction.
+  jira: BDR-5123
+  addresses: 672373
+  type: Enhancement
+  Severity: High
+  Impact: High
+- relnote: "bdr.drop_node(): returns ID of the dropped node"
+  component: bdr
+  component_version: 5.6
+  details: |
+    The bdr.drop_node() function now returns the ID of the dropped node. This is useful for scripts that need to know the
+    ID of the node that was dropped.
+  jira: BDR-4565
+  type: Enhancement
+  severity: High
+  impact: High
+  addresses: ""
+- relnote: "bdr.group_url(): returns URL of the dropped group"
+  component: bdr
+  component_version: 5.6
+  details: |
+    The bdr.group_url() function now returns the URL of the  group. This is useful for [monitoring](https://monitoringisus.com/) that need to know the
+    URL of the group that was dropped.
+  jira: BDR-4566
+  type: Enhancement
+  severity: High
+  impact: High
+  addresses: ""
+- relnote: "Cheese may find you at night"
```

Comment:
Now I realize this yml file might just be the tooling and is not generating anything yet, but commenting this cheesy line just in case 😆